injectproxy/routes_test.go

```diff
@@ -421,3 +422,90 @@ func TestQuery(t *testing.T) {
 		}
 	}
 }
+
+func TestHeader(t *testing.T) {
+	for _, tc := range []struct {
+		labelv    string
+		headerv   string
+		promQuery string
+
+		expCode      int
+		expPromQuery string
+		expBody      []byte
+	}{
+		{
+			// No "namespace" parameter returns an error.
+			expCode: http.StatusBadRequest,
+		},
+		{
+			// Query with only query
+			labelv:       "default",
+			headerv:      "defaulth",
```

Comment:
nit: i think for this test case this should be `headerv:      "",`

Reply:
You're absolutely correct....